frontend/components/panels/WalletVatDepositCheckPanel.stories.js

```diff
@@ -0,0 +1,133 @@
+import { storiesOf } from '@storybook/vue';
+import { action } from '@storybook/addon-actions';
+import faker from 'faker';
+import BigNumber from 'bignumber.js';
+import WalletVatDepositCheckPanel from './WalletVatDepositCheckPanel';
```

Comment:
Both components `WalletVatDepositCheckPanel` and `WalletDepositCheckPanel` deposit to VAT, so it's confusing naming. I propose
- `WalletVatCheckPanel` that checks
- `WalletDepositFlowCheckPanel` that also allow to execute the action